.schemas/config.schema.json

```diff
@@ -922,6 +922,50 @@
               "title": "Host",
               "description": "The network interface to listen on."
             },
+            "timeout": {
```

Comment:
This directory is no longer used - see the README :)

Reply:
Where in the readme is this mentioned?

Reply:
I should have been more specific, this one: https://github.com/ory/oathkeeper/blob/master/.schemas/README.md